stdlib/public/Platform/TiocConstants.swift

```diff
@@ -0,0 +1,131 @@
+// TIOC constants, needed only on Darwin and FreeBSD
```

Comment:
Tty IOCtl in this case, not actually an acronym in this case, TIOC* would have been more appropriate. Btw, i'm rewriting this.